CesiumUtility/include/CesiumUtility/JsonValue.h

```diff
@@ -625,32 +625,34 @@ class CESIUMUTILITY_API JsonValue final {
    */
   int64_t getSizeBytes() const noexcept {
     struct Operation {
-      int64_t operator()([[maybe_unused]] const Null& value) { return 0; }
-      int64_t operator()([[maybe_unused]] const double& value) { return 0; }
-      int64_t operator()([[maybe_unused]] const std::uint64_t& value) {
+      int64_t operator()([[maybe_unused]] const Null& /*inside*/) { return 0; }
+      int64_t operator()([[maybe_unused]] const double& /*inside*/) {
         return 0;
```

Comment:
Not in scope for this PR, just a question. What's the reason this doesn't return something like `sizeof(double)`?

Reply:
Because the size of the double is already included in the size of the variant. We only need to count extra data external to the variant.